### PR TITLE
feat: support different ssh port on different nodes

### PR DESCRIPTION
**Pull request by shjwudp**

### CODE REVIEW

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
pass CI

File: bagua/script/baguarun.py [hunk 1]
```diff
@@ -139,13 +150,23 @@ def parse_args():
 
     if args.host_list is None:
         args.host_list = os.environ.get("BAGUA_NODE_DOMAIN_NAMES", "")
-    if args.ssh_port is None:
+    if args.ssh_port is None and "BAGUA_SSH_PORT" in os.environ:
```
Comment (NOBLES5E) [inline bagua/script/baguarun.py:153]: What is `BAGUA_SSH_PORT`? KML internal variable?

Do we need to document this

Comment (shjwudp) [reply bagua/script/baguarun.py:164]: Yes, explained in argparse help.

File: bagua/script/baguarun.py [hunk 2]
```diff
@@ -87,22 +89,42 @@ def pssh_bagua_launch(
...
     parser = argparse.ArgumentParser(description="Distributed bagua launch")
-    parser.add_argument("--host_list", default=None)
-    parser.add_argument("--ssh_port", type=int, default=None)
+    parser.add_argument(
+        "--host_list", default=None,
+        help="List of target nodes, baguarun logs in to `host` via ssh. "
+        "Specify a list of nodes through host_list, which is a "
+        "comma-separated dns or ip. If the ssh port of the node is the "
+        "same, you can specify it by --ssh_port, if it is different, you "
+        "need to add: SSH_PORT after dns or ip.\n"
```
Comment (NOBLES5E) [inline bagua/script/baguarun.py:121]: add `:SSH_PORT`?

Better to give an example for easier understanding. `host1:port1,host2:port2,...`?

Comment (NOBLES5E) [reply bagua/script/baguarun.py:121]: Also after `dns` is not correct technically. should be after domain name

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

File: bagua/script/baguarun.py (hunk 2 above)
Comment (shjwudp) [reply bagua/script/baguarun.py]: DONE

**Pull request merged**